src/Stratis.Bitcoin.Features.BlockStore/BlockStoreManager.cs

```diff
@@ -6,15 +6,15 @@ public class BlockStoreManager
     {
         public IBlockRepository BlockRepository { get; }
 
-        public BlockStoreLoop BlockStoreLoop { get; }
+        public BlockStore BlockStore { get; }
 
         public IChainState ChainState { get; }
 
-        public BlockStoreManager(IBlockRepository blockRepository, IChainState chainState, BlockStoreLoop blockStoreLoop)
+        public BlockStoreManager(IBlockRepository blockRepository, IChainState chainState, BlockStore blockStore)
```

Comment:
What is this class Managing? it just appears to have 3 properties set and exposed.  I realise it wasn't actually changed in this PR,  just curious as to why it exists as an anaemic wrapper of those 3.

Reply:
I agree that this class shoud be removed but since it's used by tests and in some other places removing it will require changing quite some files. So I just don't wan't to make this PR even bigger.

Reply:
Created issue for that - https://github.com/stratisproject/StratisBitcoinFullNode/issues/1442

Reply:
nice one